### DISCUSSION TITLE
Sass @debug

### DISCUSSION THREAD

**Discussion by borisdamevin** [storybookjs/storybook#18317] [Help]
Hi,

I don't find how show the `@debug` command in console. This is possible ?

Than you.

**Comment by bozzelliandrea** [comment]
Hi @borisdamevin, debug or error is not usable in storybook, because Sass will be compiled into CSS when building the storybook.

Try to use this addon for Sass https://storybook.js.org/addons/storybook-addon-sass-postcss , but I'm not sure it will work as you wish.

**Comment by tommyc38** [comment]
Yeah, I am in the same boat trying to figure this out.
